### ISSUE TITLE
Emoticonbuttons in contributions

### ISSUE THREAD

**Issue by Nick-HC** [Human-Connection/WebApp#383]
**Is your feature request related to a problem? Please describe.**
We can choose for one emotion in one contribution. Once we've done that, we can't reverse the decision. We can only choose another emotion.

**Describe the solution you'd like**
If we chose an emotion, another click would reset the emotion and the smiley would turn gray again.

**Describe alternatives you've considered**
Disable emoticons and remove them from filters.

**Additional context**
![Issue_Emoticons](https://user-images.githubusercontent.com/47418179/54280608-2a633100-4598-11e9-9aaa-0c49ec7bfd56.jpg)

**Comment by yemiOdetola** [comment]
Can I please have more information about what needs to be done for this issue?

**Comment by roschaefer** [comment]
@blarcmann you are commenting on an issue inside an archived repository. The code in this repository has become obsolete after we migrated to [another code base](https://github.com/Human-Connection/Human-Connection).

Can you tell us how you got here? Do we have some outdated links posted somewhere?

**Comment by yemiOdetola** [comment]
Yes... But I can't really remember how I got here. I 'googled' around "open source projects I can contribute to" as a beginner and this link came up. If I can get it back, I'll update here.
